src/Microsoft.AspNetCore.Razor.Language/RazorConfiguration.cs

```diff
@@ -7,24 +7,24 @@
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    public sealed class RazorConfiguration
+    public abstract class RazorConfiguration
```

Comment:
It ended up just feeling a little better to me to take this abstract. Then if we need to flow more data in the IDE cases we can.